draft-ietf-httpbis-digest-headers.md

```diff
@@ -767,12 +766,8 @@ Specification document(s):  {{content-digest}} of this document
 
 The following examples show how representation metadata, payload transformations
 and method impacts on the message and content. When the content
-contains non-printable characters (eg. when it is compressed) it is shown as
-base64-encoded string.
-
-A request with a JSON object without any content coding.
-
-Request:
```

Comment:
@MikeBishop why remove `Request:`

Reply:
Accidentally got staged into this PR instead of #1685.

Reply:
Ok, I'll rebase soon.